modules/task_manager/cycle_routing_manager.h

```diff
@@ -64,9 +64,9 @@ class CycleRoutingManager {
  private:
   int cycle_;
   int waypoint_num_;
+  bool is_allowed_to_route_;
```

Comment:
Just In case you forget to to use init() function, please manually initialize the private data field to default values here as you use a default constructor

Reply:
OK